kubernetes/linux/main.sh

```diff
@@ -1159,6 +1159,10 @@ if [ "${GENEVA_LOGS_INTEGRATION_SERVICE_MODE}" != "true" ]; then
       sed -i -e "s/placeholder_hostname/$nodename/g" $telegrafConfFile
 fi
 
+if [ "${AZMON_RESOURCE_OPTIMIZATION_ENABLED}" == "true" ]; then
+      sed -i '/^#CustomMetricsStart/,/^#CustomMetricsEnd/ s/^/# /' $telegrafConfFile
```

Comment:
What is this exactly doing? Can you mention in the PR description exactly what changes/codepaths will be changed when AZMON_RESOURCE_OPTIMIZATION_ENABLED is true?

Reply:
right now telegraf config also has an output for custom metrics. this disables that part when resource optimization is enabled.

Reply:
Okay so this will happen after the custom metrics deprecation is announced?

Reply:
yes this will be released with custom metrics deprecation.